Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,73 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+Detail Design: Resistance System
+
+
+Function of the Subsystem: 
+The function of the resistance system is to wirelessly receive an input value from the RRS processing subsystem. The input value is to be received by the H-Bridge controller. The H-Bridge controller will send a power signal to the linear actuator in the form of a pulse width modulated value to adjust the position of the magnets across the flywheel resulting in the creation of eddy currents in the flywheel. These eddy currents create a magnetic field that opposes the source magnetic field. This causes power dissipation in the flywheel, as well as opposing torque, which creates a change of resistance felt by the user. 
+
+Constraints:
```

Comment:
The discussion of constraints is not relevant or complete. It is impossible to analyze a system that does not have clear expectations regarding the constraints and the required function.